content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.6.
+weight: 20
+---
+
+When you upgrade from Istio 1.5.x to Istio 1.6.x, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.5.x.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.5.x.
+
+Currently, Istio doesn't support skip-level upgrades. If you are using Istio 1.4, you must upgrade to Istio 1.5 first, and then upgrade to Istio 1.6. If you upgrade from versions earlier than Istio 1.4, you should first disable Galley's configuration validation. 
+
+Updated the Galley deployment:
```

Comment:
Update the Galley deployment configuration with the following steps:

Reply:
Fixed. Thanks.

Reply:
But it was a suggestion:
```suggestion
Update the Galley deployment configuration with the following steps:
```